access/templates/access/graded_form.html

```diff
@@ -8,6 +8,8 @@
 
 		{% if field.field.type == "static" %}
 		{{ field.field.more|safe }}
+    {% elif field.field.type == "hidden" %}
```

Comment:
It seems here is different indentation characters than in other places :/

Reply:
The editor didn't recognize the file being indented with tabs not spaces, my bad.